packages/template-retail-react-app/app/pages/account/wishlist/index.jsx

```diff
@@ -5,9 +5,8 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import React, {useState, useEffect, useRef} from 'react'
-import {Stack, Heading} from '@chakra-ui/layout'
```

Comment:
this is bug from `develop` branch.

Reply:
I think you might have to re-lock your package-lock files. I was seeing some errors related to this layout import.